### PR TITLE
ERD Model and table SQL

### PR DESCRIPTION
**Pull request by JamesSampson**
I have put a corrected ERD model with what I interpreted Jeff's comments to be as well as the SQL for table creation.

### CODE REVIEW

File: backpackerSQL.txt [hunk 1]
```diff
@@ -0,0 +1,95 @@
+
+CREATE TABLE Account (
+  AccountId int NOT NULL PRIMARY KEY,
```
Comment (jpuck) [inline backpackerSQL.txt:3]: @JamesSampson Put [an `AUTO_INCREMENT`](https://dev.mysql.com/doc/refman/5.7/en/example-auto-increment.html) on this and all of your other table Primary Keys too.

File: backpackerSQL.txt [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+CREATE TABLE List (
+  ListId int NOT NULL PRIMARY KEY,
+   AccountId int FOREIGN KEY references Account(AccountId),
+  name varchar(40) NOT NULL
+);
+
+CREATE TABLE GearList (
+  ListId int FOREIGN KEY references List(ListId),
+  GearId int FOREIGN KEY references Gear(GearId)
+
```
Comment (jpuck) [inline backpackerSQL.txt:55]: @JamesSampson add `PRIMARY KEY (ListId, GearId)` and the same with all your other linking tables, or else you'll end up with tons of duplicate rows.

File: backpackerSQL.txt (hunk 1 above)
Comment (JamesSampson) [reply backpackerSQL.txt:3]: Ahh yeah meant to do that! Appreciate it.

Sent from my iPhone

On Nov 19, 2016, at 10:51 PM, Jeff Puckett <notifications@github.com<mailto:notifications@github.com>> wrote:

In backpackerSQL.txthttps://github.com/russell190/backpacker/pull/9#discussion_r88793456:

> @@ -0,0 +1,95 @@
> +
> +CREATE TABLE Account (
> -  AccountId int NOT NULL PRIMARY KEY,

@JamesSampsonhttps://github.com/JamesSampson Put an AUTO_INCREMENThttps://dev.mysql.com/doc/refman/5.7/en/example-auto-increment.html on this and all of your other table Primary Keys too.

##

**Review by jpuck — COMMENTED** [review]
Strong entity types are missing auto increments and associative entity types are missing primary keys.

**Pull request merged**